### DISCUSSION TITLE
ihate plugin misses tests for import_task_choice_event

### DISCUSSION THREAD

**Discussion by linhanzhang** [beetbox/beets#4307] [Ideas]
<img width="871" alt="image" src="https://user-images.githubusercontent.com/52887992/156887654-92f99df7-0540-44aa-8a36-347e906a1018.png">

We find that the method **import_task_choice_event** in **ihate.py** does not have the test covered.

**Comment by wisp3rwind** [comment]
PRs that improve test coverage are of course warmly welcome :)

Otherwise, this is not exactly a surprise: We have far from 100% test coverage, that might not even be a realistic goal. So, unless actively doing something about it _yourself_, please don't open issues/discussions like this one. They only spam everyone who subscribes to email notifications for this project.
